src/backend/metrics/metrics.ts

```diff
@@ -268,8 +275,9 @@ export async function trackPidPlaytime(
           isBrowserGame: false,
           game_name: gameInfo.app_name,
           game_title: gameInfo.title,
-          store_name: gameInfo.runner,
+          store_name: getStoreName(gameInfo.runner),
```

Comment:
Instead of `hyperplay`, `legendary`, `nile`... it will report:
`HyperPlay`, `Epic Games`, `Amazon Games`, etc.

Should I keep this change or it will cause some reports to break?

Reply:
@awantoch

Reply:
It should be pretty easy to compile the data from the old and new formats, I think the distinctive names are solid, so I def support the change